src/components/side_panel/chart/line_chart/line_chart_design_panel.xml

```diff
@@ -0,0 +1,30 @@
+<templates>
+  <t t-name="o-spreadsheet-LineChartDesignPanel">
+    <GeneralDesignEditor t-props="props">
+      <t t-set-slot="general-extension">
+        <ChartLegend t-props="props"/>
+        <ChartShowValues t-props="props">
+          <t t-set-slot="values-extension">
```

Comment:
If we end up keeping the t-slot, use the default slot to make the tempaltes easier to read/write